tests/pytests/unit/modules/test_win_event.py

```diff
@@ -0,0 +1,134 @@
+import datetime
+
+import pytest
+
+import salt.modules.win_event as win_event
+
+pytestmark = [
+    pytest.mark.windows_whitelisted,
+    pytest.mark.skip_unless_on_windows,
+    pytest.mark.destructive_test,
```

Comment:
Why is it destructive?

Reply:
It clears the application log, then sets creates some logs in that application log, and then clears it at the end

Reply:
Marked the specific destructive tests